lua52-sys/src/lib.rs

```diff
@@ -27,6 +27,7 @@ pub const LUA_ERRERR: c_int = 6;
 pub struct lua_State;
 
 pub type lua_CFunction = extern "C" fn(L: *mut lua_State) -> c_int;
+pub type lua_CFunction2 = unsafe extern "C" fn(*mut lua_State);
```

Comment:
That still won't help the[ `luaL_requiref`](https://github.com/tomaka/hlua/pull/147/files#diff-9bfb4c4c0f0812c724dfaca35a17ddedR227) binding that needs the `lua_CFunction2` though?